src/components/AppContentListItem/AppContentListItem.vue

```diff
@@ -0,0 +1,366 @@
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<nav-element
+		class="acli_wrapper"
+		v-bind="navElement">
+		<a
+			:id="anchorId"
+			ref="acli"
+			:class="{ 'active' : isActive, 'bold': bold }"
+			href="#"
+			class="acli"
+			:aria-label="conversationLinkAriaLabel"
+			@mouseover="handleHover"
+			@focus="handleFocus"
+			@blur="handleBlur"
+			@mouseleave="handleMouseleave"
+			@keydown.tab="handleTab"
+			@click="onClick"
+			@keydown.esc="displayActions = false">
+			<!-- default slot for avatar or icon -->
+			<slot name="icon" />
+			<div class="acli-content">
+				<div class="acli-content__main">
+					<div class="acli-content__line-one">
+						<span class="acli-content__line-one__title">
+							{{ title }}
+						</span>
+						<span
+							v-if="{hasDetails}"
+							class="acli-content__line-one__details">
+							{{ details }}
+						</span>
+					</div>
+					<div class="acli-content__line-two">
+						<span v-if="hasSubtitle" class="acli-content__line-two__subtitle">
+							<slot name="subtitle" />
+						</span>
+						<span v-if="!displayActions" class="acli-content__line-two__counter">
+							<slot
+								name="counter" />
+						</span>
+					</div>
+				</div>
+				<div
+					v-show="displayActions"
+					class="acli-content__actions"
+					@click.prevent.stop="">
+					<Actions
+						ref="actions"
+						menu-align="right"
+						:aria-label="conversationSettingsAriaLabel"
+						@update:open="handleActionsUpdateOpen">
+						<slot
+							name="actions" />
+					</Actions>
+				</div>
+			</div>
+		</a>
+	</nav-element>
+</template>
+
+<script>
+import Actions from '../Actions'
+
+export default {
+	name: 'AppContentListItem',
+
+	components: {
+		Actions,
+	},
+
+	props: {
+		/**
+		 * The details text displayed in the upper right part
+		 */
+		details: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Title
+		 */
+		title: {
+			type: String,
+			required: true,
+		},
+		/**
+		* Pass in `true` if you want the matching behaviour to
+		* be non-inclusive: https://router.vuejs.org/api/#exact
+		*/
+		exact: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		* The route for the router link.
+		*/
+		to: {
+			type: [String, Object],
+			default: '',
+		},
+		/**
+		 * Id for the <a> element
+		 */
+		anchorId: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Bold title and subtitle
+		 */
+		bold: {
+			type: Boolean,
+			default: false,
+		},
```

Comment:
Used for unread rooms I assume?
Shall this be managed by the app ? Or do we want a standard for this? I'm conflicted as this seems like a very specific use case? @ChristophWurst do you need this for Mail as well? :)

Reply:
It is used in mail too, yep!